### PR TITLE
BasicHTMLPageStructure

### PR DESCRIPTION
**Pull request by AbdelrhmanAmin**
Embedding Images and Video
[Collaborative project]
This project consists of building an HTML document that matches the appearance of youtube video player page.

Navigation bar
It has an embedded image that represents a logo.
The logo is aligned to the leftmost part of the navbar.
It has an input tag that represents a search bar.
The search bar is aligned to the middle of the navbar.
It has a button tag that represents the search button.
The search button is at the end (rightmost part) of the search bar.
It has at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar.

Video
It is displayed.
It shows the video title.
It shows the number of views.
It has like and dislike icons.
It has a share button.
It shows the avatar of the user who uploaded the video.
It has a subscribe button.

Sidebar Recommended videos
It shows the recommended video’s thumbnail preview (as image not as an actual video).
It shows the title of the video.
It shows the number of views.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,429 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <title>Basic HTML Page Structure</title>
+    <link rel="stylesheet" href="src/css/style.css" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
```
Comment (descholar-ceo) [inline index.html]: It's better to use this script tag on the bottom of the body of your html content, don't use it here in head tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+  </head>
+
+  <body>
+    <nav>
+      <div class="grid-navbar">
+        <div class="logo">
+          <a style="text-decoration: none" href="#"
```
Comment (descholar-ceo) [inline index.html]: Don't use inline styling, it's better to add these stylings in your css file

File: README.md [hunk 3]
```diff
@@ -1,5 +1,48 @@
+**![](https://img.shields.io/badge/-microverse-yellow)**
+
 # Basic HTML Page Structure (Youtube Page)
 
 > This is the first project on CSS/HTML curriculum in microverse course
```
Comment (descholar-ceo) [inline README.md:5]: Try to be more descriptive on the project description

File: README.md [hunk 4]
```diff
@@ -1,5 +1,48 @@
...
+    3- Type " Code ." on CMD to open VSCode
+    4- Open with LIVE Server.
+    5- SMILE! :)
+
+## Usage
+
+      This project is made using HTML & CSS.
+      You don't need any tools to run it, Just open the index.html in your favorite browser and it's done.
+
+## Contribuiting
```
Comment (descholar-ceo) [inline README.md]: Instead of writing `Contribuiting`, you should write `Contributing` :point_right: that's a typo, fix it

File: README.md [hunk 5]
```diff
@@ -1,5 +1,48 @@
...
+
 # Basic HTML Page Structure (Youtube Page)
 
 > This is the first project on CSS/HTML curriculum in microverse course
 
+# Screenshot
+
+![screenshot](/src/screenshot_1.png)
+
+## Participants
```
Comment (descholar-ceo) [inline README.md]: On the microverse readme template, this title is `Author(s)`, if you want to customize it I would suggest using like `developers` or `contributors` this make more sense than just participant. I'd suggest you to work on this as well.

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @AbdelrhmanAmin  and @vichuge  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: You have put in enough effort to make this project, Keep it up.
## :warning: However we need to make few changes to mark this project approved, you are almost done just very few and simple remaining changes, read them below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# Required Changes
- [x] Don't use inline stylings in your HTML file, as that is mentioned by the previous TSE,
- [x] Your readme file need some more work on project specification and author section, :point_right: One author has only github link, is that okay? Maybe you can add your twitter or linkedin if you have them.

# Optional changes

- [x] Don't include scripts in head, all of scripts should be put on the bottom of the body of your html content

:point_right: Your commit messages should be in imperative mode, write something like `Add src directory and css files in it` instead of writing `Add src directory and css files in it`, refer [to this article for mor details](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

:point_right:  You have opened a lot of PRs, this is dis-couraged, never do such kind of things anymore, unless that's a requirement and next time please don't close a Pull Request which has the first review, continue working on one Pull Request untill it is approved!

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Comment by vichuge** [comment]
All the changes have been made for us.

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hello Team! 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:

### You may merge, good luck with your next projects! :rocket:
### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**